RELEASE/scripts/autoscend/iotms/clan.ash

```diff
@@ -76,7 +76,7 @@ boolean handleFaxMonster(monster enemy, boolean fightIt, string option)
 	}
 
 	auto_log_info("Faxing: " + enemy + ".", "green");
-	chat_private("cheesefax", enemy.to_string());
+	faxbot(enemy);
```

Comment:
I recall the reason we didn't use this function is it requires a chat window to be open to work. I had to manually enter chat for this function to actually fax a monster. Where as chat_private does not require users to open the chat window

perhaps it has changed in the last few months

Reply:
i don't have chat open and this worked for me, what it does happenis you never receive the message saying the fax is ready after 60s but then we just check the fax machine right after and it picks up the photocopied monster fine

Reply:
```
> ash import autoscend; autoEquip($item[combat lover\'s locket]); handleFaxMonster(428.to_monster())

[INFO] Equipping combat lover's locket to slot acc1
Preference _auto_maximize_equip_acc1 changed from to combat lover's locket
[INFO] Using fax machine to summon rock-banging cave elf
[INFO] We already have a photocopy and not the one we wanted. Disposing of bad copy.
Sending a fax.
Preference photocopyMonster changed from screambat to
You load your photocopied monster in the fax machine.
[INFO] Faxing: rock-banging cave elf.
Visiting Fax Machine in clan VIP lounge
Asking CheeseFax to send a fax of rock-banging cave elf: rock-banging cave elf
No response from CheeseFax after 60 seconds.
Countdown: 10 seconds...
Countdown: 5 seconds...
Countdown: 4 seconds...
Countdown: 3 seconds...
Countdown: 2 seconds...
Countdown: 1 second...
Waiting completed.
Receiving a fax.
You acquire an item: photocopied monster
Preference photocopyMonster changed from to rock-banging cave elf
You receive a photocopied rock-banging cave elf from the fax machine.
[INFO] Sucessfully faxed rock-banging cave elf
...
Returned: true

Reply:
ahh good call. I just saw the no response message and assumed it didn't work at all. Didn't try to see if it actually did send the fax